Cargo.toml

```diff
@@ -36,7 +36,7 @@ pretty_assertions = "1.4.0"
 rayon = "1.10.0"
 rand = "0.8.5"
 regex = "1.10.6"
-reqwest = "0.12.7"
+reqwest = { version = "0.12.7", features = ["json", "native-tls-alpn"] }
```

Comment:
we already use these featurese
https://github.com/MarcoIeni/release-plz/blob/d71a335b9c4b90d8581790d9b67259be9c9ed0f8/crates/release_plz_core/Cargo.toml#L40

This change shouldn't be necessary

Reply:
you can verify it like this:
```
$ cargo tree --format "{p} {f}" | grep reqwest
│   ├── reqwest v0.12.7 __tls,charset,default,default-tls,gzip,h2,http2,json,macos-system-configuration,native-tls,native-tls-alpn
│   ├── reqwest-middleware v0.3.3 json
│   │   ├── reqwest v0.12.7 __tls,charset,default,default-tls,gzip,h2,http2,json,macos-system-configuration,native-tls,native-tls-alpn (*)
│   ├── reqwest-retry v0.6.1 
│   │   ├── reqwest v0.12.7 __tls,charset,default,default-tls,gzip,h2,http2,json,macos-system-configuration,native-tls,native-tls-alpn (*)
│   │   ├── reqwest-middleware v0.3.3 json (*)
├── reqwest v0.12.7 __tls,charset,default,default-tls,gzip,h2,http2,json,macos-system-configuration,native-tls,native-tls-alpn (*)
```